### ISSUE TITLE
Tap on another item in the same row make a whole row flash but tap command is not invoked

### ISSUE THREAD

**Issue by NhatHo** [daniel-luberda/DLToolkit.Forms.Controls#34]
I used this code to disable row highlight. However I don't think it works, or is it just for disabling item's highlighted background color when selected?

https://github.com/daniel-luberda/DLToolkit.Forms.Controls/blob/master/Samples/iOS/Renderers/FlowListViewInternalCellRenderer.cs

The way I structured FlowListView in my code is similar to this: https://github.com/daniel-luberda/DLToolkit.Forms.Controls/blob/master/Samples/DLToolkitControlsSamples/SamplesFlowListView/SimpleGalleryPage.xaml

When users tap on the first item it responds immediately. However, 2nd tap on an item (either another item or the same item) in the same row, occasionally the row gets highlighted quickly (flash) but tap command is not called. On 3rd tap then it behaves normally. 

In ListView, I can do this:
`((ListView)sender).SelectedItem = null; // clears the 'selected' background`
This will allow me to re-select the same item immediately (without it I had to tap on another item, then I can re-select the previous item). Is there a similar technique for FlowListView?

**Comment by daniel-luberda** [comment]
You can use `FlowLastTappedItem` and `FlowLastTappedItemCommand`.

`FlowListView` uses gesture recognizers internally, here: https://github.com/daniel-luberda/DLToolkit.Forms.Controls/blob/master/FlowListView/DLToolkit.Forms.Controls.FlowListView/FlowListViewInternalCell.cs#L437-L443

Maybe you're hiding `GestureRecognizer` with some other view?

**Comment by NhatHo** [comment]
I'm using FlowItemTappedCommand and FlowLastTappedItem to get the tapped item and trigger a command on tap as follow:
`FlowItemTappedCommand="{Binding ItemTappedCommand}" FlowLastTappedItem="{Binding LastTappedItem}"`

The FlowItemTapped is an event handler, am I supposed to override it in my code to avoid the row flashing?
I changed the code to this in xaml file.
'FlowItemTappedCommand="{Binding ItemTappedCommand}" FlowLastTappedItem="{Binding LastTappedItem}" FlowItemTapped="ItemTapped"'
In xaml.cs file:
`void ItemTapped(object s, EventArgs e)
 {
       Debug.WriteLine("I am tapped here");
 }`

I commented out all of the other TapGestureRecognizer but it still only get triggered on odd number of taps (1st, 3rd, 5th, etc.) Any idea?
